src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/DataSetVersion.cs

```diff
@@ -82,7 +82,7 @@ public class DataSetVersion : ICreatedUpdatedTimestamps<DateTimeOffset, DateTime
     public DataSetVersionType VersionType
         => VersionMinor == 0 ? DataSetVersionType.Major : DataSetVersionType.Minor;
 
-    public bool CanBeDeleted => Status is DataSetVersionStatus.Failed
+    public bool CanBeUpdated => Status is DataSetVersionStatus.Failed
         or DataSetVersionStatus.Mapping
         or DataSetVersionStatus.Draft
         or DataSetVersionStatus.Cancelled;
```

Comment:
Should we allow updating changelong notes on a data set version that's failed or cancelled?

Reply:
We've now discussed this and agreed to only allow it on `Draft` and `Mapping` :)